packages/frontend-2/components/settings/workspaces/security/sso/DeleteDialog.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <LayoutDialog
+    v-model:open="isOpen"
+    title="Remove SSO Provider"
+    max-width="sm"
+    :buttons="dialogButtons"
+  >
+    <p class="text-body-xs text-foreground">
+      Are you sure you want to remove
+      <span class="font-semibold">{{ providerName }}</span>
+      from your workspace's SSO configuration?
+    </p>
+    <p class="text-body-xs text-foreground-2 my-2">
+      Users who have used SSO will no longer be able to sign in using their
+      organization's credentials.
+    </p>
+  </LayoutDialog>
+</template>
+
+<script setup lang="ts">
+import type { LayoutDialogButton } from '@speckle/ui-components'
+import { useWorkspaceSsoDelete } from '~/lib/workspaces/composables/management'
+
+const props = defineProps<{
+  providerName: string
+  workspaceSlug: string
+}>()
+
+const emit = defineEmits<{
+  (e: 'deleted'): void
+}>()
+
+const isOpen = defineModel<boolean>('open', { required: true })
+const { deleteSsoProvider } = useWorkspaceSsoDelete()
+
+const handleRemove = async () => {
+  const success = await deleteSsoProvider(props.workspaceSlug)
+  if (success) {
```

Comment:
What happens when it doesn't succeed? Maybe trigger a toast on both succes and error

Reply:
The toasts are triggered in useWorkspaceSsoDelete